src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
+      -webkit-box-pack: center;
```

Comment:
Why do you need `-webkit-box-pack: center;`?

Reply:
On referring 

-  https://www.youtube.com/watch?v=B-MNkBh7F2A . 
-  menu.scss

I created -WebKit-box and relevant changes

Removing it now.